packages/Bio/detectors.js

```diff
@@ -109,7 +110,8 @@ class BioPackageDetectors extends DG.Package {
       }
 
       const decoyAlphabets = [
-        ['SMILES', this.SmilesRawAlphabet, 0.30],
+        ['NUMBERS', this.NumbersRawAlphabet, 0.25],
+        ['SMILES', this.SmilesRawAlphabet, 0.25],
```

Comment:
What are these numbers?

Reply:
These values are chosen empirically to correctly classify according to the test data samples.